docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+{
```

Comment:
<blockquote>Sometimes, risks can affect other risks, which is a concept known in GBD as "mediation."</blockquote><p>It's known as mediation <em>outside</em> GBD as well, right? Seems like we should indicate that this is a standard term in epidemiology, etc., rather than something specific to GBD.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>